.github/workflows/ci.yml

```diff
@@ -7,21 +7,30 @@ jobs:
     name: lint
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.19
+      - uses: actions/checkout@v3
       - name: golangci-lint
-        uses: golangci/golangci-lint-action@v2
+        uses: golangci/golangci-lint-action@v3
         with:
-          version: v1.29
+          version: latest
```

Comment:
I think in past projects we've pinned to a specific version to avoid sudden breakage when there hasn't been any code change, but a new release of the linter pulls in new checks. Just mentioning here in case we do run in to this situation and want to go back to a specific version.

Reply:
Hmmm true. We could just pin to the current latest and attempt to keep it updated?